### DISCUSSION TITLE
飞书分块排版中的内容无法被获取

### DISCUSSION THREAD

**Discussion by xiehuacheng** [LetTTGACO/elog#91] [General]
对于飞书中这样分块排版的内容，转出来的文章中直接缺失了这些部分
<img width="803" alt="image" src="https://github.com/LetTTGACO/elog/assets/52902182/fb4b10a0-0c65-4b50-a8d8-5fa7ac135f98">

**Comment by xiehuacheng** [comment]
正常的竖排的内容是正常的

**Comment by LetTTGACO** [comment]
markdown 无法识别分栏，你只能使用 markdown 兼容的语法，详情可以查看这里的飞书示例文档
https://elog.1874.cool/feishu/VULCdSLgxotcb1xLi1BcWwdPnVa

**Reply by xiehuacheng** [reply]
好的，感谢作者
